components/buttons/Instruction.js

```diff
@@ -5,12 +5,19 @@ function Instruction() {
     const divBoard = document.getElementById('divBoard');
     const instruction = document.getElementById('InstructionCard');
     const keyboard = document.getElementById('Keyboard');
+    const setUnderBoard = document.getElementById('divUnderBoard')
+    const flags = document.getElementById('flags');
```

Comment:
Spróbuj zacząć odchodzić od selektorów na korzyść zaimplementowania tego za pomocą Reactu.

useRef i useState

https://stackoverflow.com/questions/59198952/using-document-queryselector-in-react-should-i-use-refs-instead-how

https://www.upbeatcode.com/react/how-to-use-queryselector-in-react/